src/main/java/com/bfwg/config/WebSecurityConfig.java

```diff
@@ -56,46 +58,45 @@ public AuthenticationManager authenticationManager(AuthenticationConfiguration a
 
 	@Bean
 	public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
+
 		http
-				.sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS).and()
-				.exceptionHandling().authenticationEntryPoint(restAuthenticationEntryPoint).and()
-				.authorizeRequests()
-				.antMatchers(
-						HttpMethod.GET,
-						"/",
-						"/auth/**",
-						"/webjars/**",
-						"/*.html",
-						"/favicon.ico",
-						"/**/*.html",
-						"/**/*.css",
-						"/**/*.js")
-				.permitAll()
-				.antMatchers("/auth/**").permitAll()
-				.anyRequest().authenticated().and()
 				.addFilterBefore(new TokenAuthenticationFilter(tokenHelper, jwtUserDetailsService),
-						BasicAuthenticationFilter.class);
+						BasicAuthenticationFilter.class)
+				.authorizeHttpRequests(authorize -> authorize
+						.requestMatchers(
+								antMatcher(HttpMethod.GET, "/"),
```

Comment:
Do you want to whitelist swagger endpoints as well in case we add support for that?

Reply:
I think it's better to add any swagger configuration only when swagger will be eventually added, so the starter project stays as clean as possible.